### DISCUSSION TITLE
مشکل

### DISCUSSION THREAD

**Discussion by jpro123** [evryn/laravel-toman#31] [Q&A]
سلام

موقع ارسال درخواست خطای زیر را نمایش می دهد  . 

 Method Evryn\LaravelToman\PendingRequest::orderId does not exist. 

متد orderId در فایل PendingRequest یافت نمی شود . 

در فایل مذکور هم که نگاه میکنم همچنین متدی وجود ندارد . 

لطف اصلاح کنین .

**Comment by AmirrezaNasiri** [comment]
سلام، وقت بخیر
عذر میخوام بابت این تاخیر در پاسخگویی. احتمالا دارین از درایور زرین‌پال استفاده میکنین. ‌orderId توی این درایور وجود نداره و مال درگاه IDPay هست.
